### ISSUE TITLE
Request: Storj

### ISSUE THREAD

**Issue by CodeShakingSheep** [simple-icons/simple-icons#11126]
### Brand Name

Storj

### Website

https://www.storj.io/

### Popularity Metric

The Similarweb rank is 283,503. See https://www.similarweb.com/website/storj.io

### Official Resources for Icon and Color

- SVG: https://www.storj.io/company/media-kit
- Color: `#0149FF`

### Additional Comments

> 6.   TRADEMARKS
‍“Storj,” “Storj Labs,” “Storj DCS” the Company logo, any other Company service names, logos, mascots or slogans that may appear on the Services, and the look and feel of the Services, including all page headers, custom graphics, button icons and scripts, are trademarks, service marks or trade dress of Company and our suppliers and our licensors, and, except as otherwise authorized in the Agreement or any open source license associated with the Services, may not be copied, imitated or used, in whole or in part, without our or the applicable trademark holder’s prior written permission. You may not use any metatags or other “hidden text” utilizing “Storj Labs” or any other name, trademark or product or service name of Company without our prior written permission. Further, you may not use, frame or utilize framing techniques to enclose any Company trademark, logo or other proprietary information, including the images found on the Services, the content of any text or the layout or design of any page, or form contained on a page, on the Services without our express written consent.

I think this might require asking for permission.

**Comment by adamrusted** [comment]
Yep, you're absolutely right that this will require permission. Would you like to reach out to them @CodeShakingSheep?

**Comment by CodeShakingSheep** [comment]
> Yep, you're absolutely right that this will require permission. Would you like to reach out to them @CodeShakingSheep?

I just asked for permission via mail.

**Comment by PeterShaggyNoble** [comment]
See #13453
